e2e/tests/functional/plugins/notebook/notebookWithCouchDB.e2e.spec.js

```diff
@@ -0,0 +1,182 @@
+        await page.locator('[aria-label="Notebook Entry Input"] >> nth=1').click();
+        await page.locator('[aria-label="Notebook Entry Input"] >> nth=1').fill(`Second Entry`);
+
+        await page.locator('text=To start a new entry, click here or drag and drop any object').click();
+        await page.locator('[aria-label="Notebook Entry Input"] >> nth=2').click();
+        await page.locator('[aria-label="Notebook Entry Input"] >> nth=2').fill(`Third Entry`);
+
+        // Add three tags
+        await page.hover(`button:has-text("Add Tag") >> nth=2`);
+        await page.locator(`button:has-text("Add Tag") >> nth=2`).click();
+        await page.locator('[placeholder="Type to select tag"]').click();
+        await page.locator('[aria-label="Autocomplete Options"] >> text=Science').click();
+        await page.waitForSelector('.c-tag__label:has-text("Science")');
+
+        await page.hover(`button:has-text("Add Tag") >> nth=2`);
+        await page.locator(`button:has-text("Add Tag") >> nth=2`).click();
+        await page.locator('[placeholder="Type to select tag"]').click();
+        await page.locator('[aria-label="Autocomplete Options"] >> text=Drilling').click();
+        await page.waitForSelector('.c-tag__label:has-text("Drilling")');
+
+        await page.hover(`button:has-text("Add Tag") >> nth=2`);
+        await page.locator(`button:has-text("Add Tag") >> nth=2`).click();
+        await page.locator('[placeholder="Type to select tag"]').click();
+        await page.locator('[aria-label="Autocomplete Options"] >> text=Driving').click();
+        await page.waitForSelector('.c-tag__label:has-text("Driving")');
+        page.waitForLoadState('networkidle');
+
+        // Add a fourth entry
+        requests = [];
+        await page.locator('text=To start a new entry, click here or drag and drop any object').click();
+        await page.locator('[aria-label="Notebook Entry Input"] >> nth=3').click();
+        await page.locator('[aria-label="Notebook Entry Input"] >> nth=3').fill(`Fourth Entry`);
+        page.waitForLoadState('networkidle');
+
+        expect(requests.length).toBeLessThanOrEqual(5);
+
+        // Add a fifth entry
+        requests = [];
+        await page.locator('text=To start a new entry, click here or drag and drop any object').click();
+        await page.locator('[aria-label="Notebook Entry Input"] >> nth=4').click();
+        await page.locator('[aria-label="Notebook Entry Input"] >> nth=4').fill(`Fifth Entry`);
+        page.waitForLoadState('networkidle');
+
+        expect(requests.length).toBeLessThanOrEqual(5);
+
+        // Add a sixth entry
+        requests = [];
+        await page.locator('text=To start a new entry, click here or drag and drop any object').click();
+        await page.locator('[aria-label="Notebook Entry Input"] >> nth=5').click();
+        await page.locator('[aria-label="Notebook Entry Input"] >> nth=5').fill(`Sixth Entry`);
```

Comment:
This will only fill the entry text box, but not submit it. Is this intended? We would need to hit keyboard 'Enter' or click off in order to submit the change I believe?

Reply:
You're right!